paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/FormViewModel.kt

```diff
@@ -50,18 +51,21 @@ internal class FormViewModel @Inject internal constructor(
     internal class Factory(
         val config: FormFragmentArguments,
         val resource: Resources,
-        var layout: LayoutSpec
+        var layout: LayoutSpec,
+        private val contextSupplier: () -> Context
     ) : ViewModelProvider.Factory, Injectable<Factory.FallbackInitializeParam> {
         internal data class FallbackInitializeParam(
-            val resource: Resources
+            val resource: Resources,
+            val context: Context
         )
 
         @Inject
         lateinit var subComponentBuilderProvider: Provider<FormViewModelSubcomponent.Builder>
 
         @Suppress("UNCHECKED_CAST")
         override fun <T : ViewModel?> create(modelClass: Class<T>): T {
-            injectWithFallback(config.injectorKey, FallbackInitializeParam(resource))
+            val context = contextSupplier()
+            injectWithFallback(config.injectorKey, FallbackInitializeParam(resource, context))
```

Comment:
nit: you can probably just call contextSupplier() without saving it to a variable.

Reply:
Android Studio yells at me if I try that

Reply:
weird, I am not seeing that, curious what the error is, but ok if it can't be fixed.

Reply:
I get 
`Return type is 'T#1 (type parameter of com.stripe.android.paymentsheet.forms.FormViewModel.Factory.create)', which is not a subtype of overridden public abstract fun <T : ViewModel!> create(modelClass: Class<(T#2 (type parameter of androidx.lifecycle.ViewModelProvider.Factory.create)..T#2?)>): T#2!! defined in androidx.lifecycle.ViewModelProvider.Factory`

Reply:
sorry, just want to make sure we are talking about the same thing.   Are you doing this on line 68?:
```
injectWithFallback(
                config.injectorKey,
                FallbackInitializeParam(resource, contextSupplier())
            )
```